packages/sdk/src/constants.ts

```diff
@@ -1,2 +1,12 @@
+import { SupportedTokens } from '@sovryn/contracts';
+
 // Default slippage for swap, 10_000 = 100%.
 export const DEFAULT_SWAP_SLIPPAGE = 100; // 1%
+
+export const stableCoins: SupportedTokens[] = [
```

Comment:
```suggestion
export const STABLECOINS: SupportedTokens[] = [
```
It needs to be uppercased to follow our guidelines.